app/controllers/miq_ae_tools_controller.rb

```diff
@@ -203,7 +203,7 @@ def upload
   def export_datastore
     filename = "datastore_" + format_timezone(Time.now, Time.zone, "fname") + ".zip"
     disable_client_cache
-    send_data(MiqAeDatastore.export,
+    send_data(MiqAeDatastore.export(current_tenant),
               :filename => filename)
```

Comment:
@kbrock 
Thats how it was, I just added the current_tenant parameter, will change.